app/views/carts/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<ul>
+<% @items.each do |item| %>
+    <li>Item: <%= item.title %><br/>
+    Description: <%= item.description %><br/>
+    Price: $<%= item.price %>
+<%# Add thumbnail image %>
+<%# image_tag(item.image.thumbnail) %>
+    </li>
+<% end %>
+<p>Total Price of Cart: $<%= @items.map { |item| item.price }.reduce(:+) %>
```

Comment:
Like you mentioned in person, we can move this logic to models. We can work on that together.